core/os/keyboard.h

```diff
@@ -100,54 +119,54 @@ enum class Key {
 	KP_7 = SPECIAL | 0x8D,
 	KP_8 = SPECIAL | 0x8E,
 	KP_9 = SPECIAL | 0x8F,
-	SUPER_L = SPECIAL | 0x2C,
-	SUPER_R = SPECIAL | 0x2D,
-	MENU = SPECIAL | 0x2E,
-	HYPER_L = SPECIAL | 0x2F,
-	HYPER_R = SPECIAL | 0x30,
-	HELP = SPECIAL | 0x31,
-	DIRECTION_L = SPECIAL | 0x32,
-	DIRECTION_R = SPECIAL | 0x33,
-	BACK = SPECIAL | 0x40,
-	FORWARD = SPECIAL | 0x41,
-	STOP = SPECIAL | 0x42,
-	REFRESH = SPECIAL | 0x43,
-	VOLUMEDOWN = SPECIAL | 0x44,
-	VOLUMEMUTE = SPECIAL | 0x45,
-	VOLUMEUP = SPECIAL | 0x46,
-	BASSBOOST = SPECIAL | 0x47,
-	BASSUP = SPECIAL | 0x48,
-	BASSDOWN = SPECIAL | 0x49,
-	TREBLEUP = SPECIAL | 0x4A,
-	TREBLEDOWN = SPECIAL | 0x4B,
-	MEDIAPLAY = SPECIAL | 0x4C,
-	MEDIASTOP = SPECIAL | 0x4D,
-	MEDIAPREVIOUS = SPECIAL | 0x4E,
-	MEDIANEXT = SPECIAL | 0x4F,
-	MEDIARECORD = SPECIAL | 0x50,
-	HOMEPAGE = SPECIAL | 0x51,
-	FAVORITES = SPECIAL | 0x52,
-	SEARCH = SPECIAL | 0x53,
-	STANDBY = SPECIAL | 0x54,
-	OPENURL = SPECIAL | 0x55,
-	LAUNCHMAIL = SPECIAL | 0x56,
-	LAUNCHMEDIA = SPECIAL | 0x57,
-	LAUNCH0 = SPECIAL | 0x58,
-	LAUNCH1 = SPECIAL | 0x59,
-	LAUNCH2 = SPECIAL | 0x5A,
-	LAUNCH3 = SPECIAL | 0x5B,
-	LAUNCH4 = SPECIAL | 0x5C,
-	LAUNCH5 = SPECIAL | 0x5D,
-	LAUNCH6 = SPECIAL | 0x5E,
-	LAUNCH7 = SPECIAL | 0x5F,
-	LAUNCH8 = SPECIAL | 0x60,
-	LAUNCH9 = SPECIAL | 0x61,
-	LAUNCHA = SPECIAL | 0x62,
-	LAUNCHB = SPECIAL | 0x63,
-	LAUNCHC = SPECIAL | 0x64,
-	LAUNCHD = SPECIAL | 0x65,
-	LAUNCHE = SPECIAL | 0x66,
-	LAUNCHF = SPECIAL | 0x67,
+	SUPER_L = SPECIAL | 0x3F,
```

Comment:
I'm confused by this change, doesn't it break support for all those named special keys?

Reply:
As I understand (which can be wrong of course) no, but honestly I'm not too sure about it.

Reply:
It seems fine, but it's probably better to add new keys to the free codes, instead of shifting everything. Just in case some code is using raw values, not constants.

Reply:
> It seems fine, but it's probably better to add new keys to the free codes, instead of shifting everything. Just in case some code is using raw values, not constants.

yes, you're right, I thought for the sake of consistency it's better to use F key values in order, maybe that was a risky idea :)

Reply:
Could maybe start at `0x40` and end at `0x6F` instead of `0x3F`/`0x6E`?

Reply:
I was confused about it as hell :D

in the Key class, the escape's value is 0x01
on my fedora 36 (with wayland) as I started checking the keycodes I got 0x01
then in the key_mapping_x11.cpp I found 0x09
in the windows-related keymapping.cpp it's 0x01 again
in the osx related mm file it's a third one

Reply:
> Could maybe start at `0x40` and end at `0x6F` instead of `0x3F`/`0x6E`?

sure why not :)